rust/crd/src/affinity.rs

```diff
@@ -60,7 +62,16 @@ mod tests {
                 replicas: 1
         "#;
         let hive: HiveCluster = serde_yaml::from_str(input).expect("illegal test input");
-        let merged_config = hive.merged_config(&HiveRole::MetaStore, "default").unwrap();
+
+        let rolegroup_ref = RoleGroupRef {
+            cluster: ObjectRef::from_obj(&hive),
+            role: HiveRole::MetaStore.to_string(),
+            role_group: "default".to_string(),
+        };
+
+        let merged_config = hive
+            .merged_config(&HiveRole::MetaStore, &rolegroup_ref)
+            .unwrap();
```

Comment:
I think this could be improved: https://github.com/stackabletech/trino-operator/blob/3ba23121259ccfea9e336974ca8cb4ed939b73ec/rust/crd/src/affinity.rs#L126-L128C18

Reply:
Improved in 325ae1a